flytekit/clients/auth/auth_client.py

```diff
@@ -303,6 +303,7 @@ def _credentials_from_response(self, auth_token_resp) -> Credentials:
         """
         response_body = auth_token_resp.json()
         refresh_token = None
+        expires_in = None
```

Comment:
There was a warning about this possibly not being defined in my editor due to the if statement below.